app/controllers/preprints/provider/preprint-detail.js

```diff
@@ -64,6 +65,7 @@ export default Controller.extend({
         const { location: { origin } } = window;
         return [
             origin,
+            this.get('theme.id') !== 'osf' ? `preprints/${this.get('theme.id')}` : null,
```

Comment:
For future reference, this route is defined [here](https://github.com/CenterForOpenScience/osf.io/blob/76fb0c739f4cdabf03b5bfd2bc63d83b1c2d4796/website/routes.py#L279-L290) in `osf.io` code